dubbo-common/src/main/java/org/apache/dubbo/common/constants/CommonConstants.java

```diff
@@ -600,4 +600,8 @@ public interface CommonConstants {
      *
      */
     String DUBBO_JSTACK_MAXLINE = "dubbo.jstack-dump.max-line";
+
+
+    String ENCODE_IN_IO_THREAD_KEY = "encode.in.io";
+    boolean DEFAULT_ENCODE_IN_IO_THREAD = true;
```

Comment:
why not set `false` as default

Reply:
I think this is a new feature, maybe use `false` as default

Reply:
If this doesn't break usage for some users, we can enable it by default. By the way, this PR will be merged into the `3.2` branch, which can accept some breaking changes.

Reply:
OK , get